### PR TITLE
Translate 2014-10-27-rexml-dos-cve-2014-8080 (id)

### PR DESCRIPTION
**Pull request by meisyal**
Hello all,

This is my first time translating www.ruby-lang.org post into Bahasa Indonesia. I have seen there were so many untranslated posts in Bahasa Indonesia since October 2014. So, I decided to help translate them in order to catch up English version of www.ruby-lang.org. I hope I still have a time to translate the rest.

Thank you.
meisyal.

### CODE REVIEW

**Comment by hlcfan** [comment]
Thanks for your help. Could you please add the language to the title of this PR to make it more clear for reviewing?

**Comment by meisyal** [comment]
Thanks. I have edited the title of this PR.

File: id/news/_posts/2014-10-27-rexml-dos-cve-2014-8080.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+[CVE-2014-8080](http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2014-8080).
+Kami sangat merekomendasikan untuk memperbarui versi Ruby.
+
+## Detil
+
+Ketika membaca ujung teks sebuah dokumen XML, REXML *parser* dapat
+dipaksa untuk mengalokasikan objek string yang sangat besar yang mana dapat
+mengonsumsi keseluruhan memori dalam sebuah mesin, menyebabkan *denial of service*.
+
+Imbas tersebut akan kelihatan seperti kode berikut:
```
Comment (gozali) [inline id/news/_posts/2014-10-27-rexml-dos-cve-2014-8080.md]: can you change 'kelihatan' to 'terlihat'

**Comment by gozali** [comment]
@meisyal commented on line 22, otherwise +1, thanks :+1:

**Comment by meisyal** [comment]
@gozali I have edited.

**Comment by gozali** [comment]
@meisyal cool, thanks

**Comment by stomar** [comment]
Please squash into a **single commit** (`Translate 2014-10-27-rexml-dos-cve-2014-8080 (id)`) and force push your topic branch.

**Comment by meisyal** [comment]
@stomer I have done. Please, review my latest commit!
Thanks.

**Comment by stomar** [comment]
Thanks! Merged manually as eeef749bd6ee41cc842f80a4d2dc5e1ab5df276f. cc @gozali

**Pull request closed without merging**